examples/ray/README.md

```diff
@@ -16,6 +16,12 @@ Or run locally
 torchx run -s ray -cfg working_dir=~/repos/torchrec/examples/ray,requirements=./requirements.txt dist.ddp -j 1x2 --script ~/repos/torchrec/examples/ray/train_torchrec.py
 ```
 
+To run with GPUs, add --gpus as a flag to the dist.ddp component
+For other dist.ddp options, see https://pytorch.org/torchx/latest/components/distributed.html
+```
+torchx run -s ray -cfg working_dir=~/repos/torchrec/examples/ray,requirements=./requirements.txt dist.ddp -j 1x2 --gpu 2 --script ~/repos/torchrec/examples/ray/train_torchrec.py
```

Comment:
In this case, `--gpu 2` means that one worker has 2 GPUs or 1 node has 2 GPUs?

Reply:
ok, `--gpu` is gpus per replica, so if I want to train 1 node with 2 GPUs, the config should be 
```
dist.ddp -j 1x2 --gpu 1
```
?

Reply:
this is the number of GPUs per replica (ie per node or per host). so, it would be the same as the number after the x (i.e -j 1x2 --gpu 2, or -j 2x4 --gpu 4)